index.js

```diff
@@ -82,6 +86,18 @@ export const hpe = deepFreeze({
     },
     extend: 'letter-spacing: 0.04167em;',
   },
+  checkBox: {
+    icon: {
+      extend: css`
```

Comment:
I saw this in the `grommet` theme too. Why can't we put this in `base`?

Reply:
because growing the theme to take the full width of the box seems specific to us. Our theme does not want that. it is centered, as most checkboxes. it was hard to us to customize it with this there.